docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -1,7 +1,7 @@
 [[configure-endpoint-integration-policy]]
 = Configure the {endpoint-sec} integration policy
 
-After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum or Enterprise license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, and trusted applications to meet your organization's security needs.
+After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum or Enterprise license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, trusted applications, event filters, and host isolation exceptions to meet your organization's security needs.
```

Comment:
`are automatically enabled on protected hosts (a Platinum or Enterprise license feature)`

We should maybe move or delete the note about Platinum or Enterprise, since not all protections in a policy are platinum/enterprise (Ransomware, Memory Threat, and Malicious Behavior are the paid protections)

Reply:
@caitlinbetz How about a revision? I'm hesitant to delete it, since otherwise the reader might not realize some of these features aren't free. We're pretty clear about licensing when we discuss each feature individually further down the page, but a blanket statement at the top might help set expectations.
```suggestion
After the {agent} is installed with the {endpoint-sec} integration, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (some features require a Platinum or Enterprise license). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, trusted applications, event filters, and host isolation exceptions to meet your organization's security needs.
```

Reply:
Yeah agreed! Just worried that it was confusing before. This looks good!